client/src/components/Sadness.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components/macro";
+import smile from "../assets/icon/smile.svg";
+import sad from "../assets/icon/coloredSad.svg";
+import { Link } from "react-router-dom";
+
+const Header = styled.h1`
+  font-family: Roboto;
+  font-weight: 400;
+  position: fixed;
+  top: 25%;
+  display: grid;
+  place-self: left;
+`;
+
+const Text = styled.h2`
+  position: fixed;
+  top: 39%;
+  display: grid;
+  place-self: left;
+`;
+
+const Container = styled.div`
+  position: fixed;
+  top: 57%;
+`;
+
+const Happiness = styled.img`
+  margin-right: 10px;
+`;
+
+const Sadness = styled.img`
+  margin-left: 10px;
+`;
+
+export default function HappyComp() {
+  return (
+    <>
+      <Header>Das ist ok!</Header>
+      <Text>Gönn’ dir heute viel Ruhe.</Text>
+      <Container>
+        <Link to="/">
+          <Happiness src={smile} alt="happy" />
+          <Sadness src={sad} alt="sad" />
+        </Link>
+      </Container>
```

Comment:
This and the Happiness component only seem to differentiate in the text. You could build a single component and pass the text via props

Reply:
@phlgr Thanks for your review. But I'm going back and forth with it. I can not find a solution to change the text elements by clicking the "Happiness" or the "Sadness" button. Do you have more information to add?